tests/dtm/test_multi_m0d_process_restart.py

```diff
@@ -303,3 +303,345 @@ def test_delete_after_multi_m0d_restart(self):
+                                      obj_size=self.test_cfg['size'],
+                                      log_file_prefix=log_file_prefix, queue=que)
+        resp = que.get()
+        assert_utils.assert_true(resp[0], resp[1])
+        workload_info_write = resp[1]
+
+        self.log.info("Step 2: Start READ Operations in loop in background for data written in "
+                      "step 1:")
+        proc_read_op = multiprocessing.Process(target=self.dtm_obj.perform_ops,
+                                               args=(workload_info_write, que, False, True,
+                                                     True, self.test_cfg['loop_count']))
+        proc_read_op.start()
+        time.sleep(self.delay)
+
+        self.log.info("Step 3: Perform m0d Process Restart on %s unique data pods during"
+                      "Read Operations at same time",self.kvalue)
+        pods_list = []
+        m0d_proc_list = []
+        count = 0
+        while count < self.kvalue:
+            pod_selected, container = self.master_node_list[0].select_random_pod_container(
+                const.POD_NAME_PREFIX, const.MOTR_CONTAINER_PREFIX)
+            if pod_selected in pods_list:
+                continue
+            else:
+                pods_list.append(pod_selected)
+                count = count+1
+            proc = multiprocessing.Process(target = self.dtm_obj.process_restart_mp,
+                                           args = (self.master_node_list[0],pod_selected,container,
+                                                   self.m0d_process, 1200, m0d_que))
+            proc.start()
+            m0d_proc_list.append(m0d_proc_list)
+
+        self.log.info("Sleep for 1200+120 seconds")
+        time.sleep(1320)
+
+        self.log.info("Verify if all the processes recovered")
+        for each in m0d_proc_list:
+            if each.is_alive():
+                each.join()
+            resp = m0d_que.get()
+            assert_utils.assert_true(resp, "Failure during m0d process restart/recovery")
+
+        self.log.info("Step 5: Verify hctl status if all services are online")
+        resp = self.ha_obj.poll_cluster_status(self.master_node_list[0], 300)
+        assert_utils.assert_true(resp[0],resp[1])
+
+        self.log.info("Step 6a: Wait for READ Operation to complete.")
+        if proc_read_op.is_alive():
+            proc_read_op.join()
+        resp = que.get()
+        assert_utils.assert_true(resp[0], resp[1])
+
+        self.test_completed = True
+        self.log.info("ENDED:  Verify continuous WRITE and READ during different m0d restart "
+                      " using pkill")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-47163")
+    def test_continuous_write_read_during_k_m0d_restart(self):
+        """
+        Verify Continuous Write/Reads during K process restart.
+        (Randomly select m0d process, kill it ,sleep for 5 mins, start it and.Repeat this K times)
```

Comment:
so we have same process getting killed here if diff then the only diff between 1st and this is, in 1st we are killing processes after every 2 mins and restart after 5 and here its one by one. Is that right?